src/hotspot/cpu/aarch64/methodHandles_aarch64.cpp

```diff
@@ -34,6 +34,7 @@
 #include "prims/methodHandles.hpp"
 #include "runtime/flags/flagSetting.hpp"
 #include "runtime/frame.inline.hpp"
+#include "runtime/runtimeUpcalls.hpp"
```

Comment:
Is there any code in this file that requires this header?

Reply:
nope, thank you